components/Dialogs/src/MDCAlertController.h

```diff
@@ -69,6 +69,24 @@
  */
 @property(nonatomic, nonnull, readonly) NSArray<MDCAlertAction *> *actions;
 
+/** The font applied to the title of Alert Controller.*/
+@property(nonatomic, strong, nullable) UIFont *titleFont;
+
+/** The color applied to the title of Alert Controller.*/
+@property(nonatomic, strong, nullable) UIColor *titleColor;
+
+/** The font applied to the message of Alert Controller.*/
+@property(nonatomic, strong, nullable) UIFont *messageFont;
+
+/** The color applied to the message of Alert Controller.*/
+@property(nonatomic, strong, nullable) UIColor *messageColor;
+
+/** The font applied to the button of Alert Controller.*/
+@property(nonatomic, strong, nullable) UIFont *buttonFont;
+
+/** The color applied to the button of Alert Controller.*/
+@property(nonatomic, strong, nullable) UIColor *buttonColor;
```

Comment:
Is this the button background color or text color?
Do we need to expose both?

Reply:
This actually is confusing to me so I only exposed what was already exposed.
This actually could get complicated, since buttons have colors for different state. Like setting the background color, sets it in normal state but what about other states such as selected or highlighted. I am gonna create a separate issue to address this and change the property name to make it more clear.

Reply:
Pivotal issue https://www.pivotaltracker.com/story/show/156540544